__tests__/app.test.js

```diff
@@ -169,35 +168,95 @@ describe("/api/articles/:article_id/comments", () => {
       .post("/api/articles/100/comments")
       .send({
         username: "butter_bridge",
-        body: "This is a test comment"
+        body: "This is a test comment",
       })
       .expect(404)
       .then(({ body }) => {
         expect(body.message).toBe("not found");
       });
-  })
-  it('POST 400: returns error for bad id', () => {
+  });
+  it("POST 400: returns error for bad id", () => {
     return request(app)
-     .post("/api/articles/dog/comments")
-     .send({
+      .post("/api/articles/dog/comments")
+      .send({
         username: "butter_bridge",
-        body: "This is a test comment"
+        body: "This is a test comment",
       })
       .expect(400)
       .then(({ body }) => {
         expect(body.message).toBe("bad request");
       });
   });
-  it('POST 400: returns error for posting incorrect column names', () => {
+  it("POST 400: returns error for posting incorrect column names", () => {
     return request(app)
-     .post("/api/articles/1/comments")
-     .send({
+      .post("/api/articles/1/comments")
+      .send({
         banana: "butter_bridge",
-        body: "This is a test comment"
+        body: "This is a test comment",
       })
       .expect(400)
       .then(({ body }) => {
         expect(body.message).toBe("bad request");
       });
   });
+});
+
+describe("/api/articles/:article_id", () => {
+  it("PATCH 200: updates the vote of an article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1000 })
+      .expect(200)
+      .then(({ body: { article } }) => {
+        console.log(article);
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 1100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  it("PATCH 404: returns error for non existing id", () => {
+    return request(app)
+      .patch("/api/articles/100")
+      .send({ inc_votes: 1000 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("not found");
+      });
+  
+})
+it('PATCH 400: returns error for bad id', () => {
+    return request(app)
+     .patch("/api/articles/dog")
+     .send({ inc_votes: 1000 })
+     .expect(400)
+     .then(({ body }) => {
+        expect(body.message).toBe("bad request");
+      });
+});
+it('PATCH 400: returns error for invalid request body', () => {
+    return request(app)
+     .patch("/api/articles/1")
+     .send({ inc_votes: "banana" })
+     .expect(400)
+     .then(({ body }) => {
+        expect(body.message).toBe("bad request");
+      });
+});
+it('PATCH 400: returns error for invalid request body', () => {
```

Comment:
Does this test differ from the previous one? If so lets update the description to show the difference. If it's functionally the same and auto passed maybe we don't need it.